File: java/1002.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        double raio, area;
        final double PI = 3.14159;
        Scanner input = new Scanner(System.in);
        
        raio = input.nextDouble();
        
        area = PI * (raio * raio);
        
        System.out.printf("A=%.4f\n", area);
    }
}

File: src/c/1002.c
#include <stdio.h>

int main() {
    double area;
    double raio;
    const double N = 3.14159;

	scanf("%lf", &raio);

    area = N * (raio * raio);

	printf("A=%.4lf\n", area);

    return 0;
}
